app/Http/Controllers/PortalsController.php

```diff
@@ -105,7 +105,9 @@ public function importUsers(Request $request): JsonResponse
                         $abbreviation = 'ET';
                     } elseif ($abbreviation == 'ET-MASTER') {
                         $abbreviation = 'ET';
-                    } elseif ($abbreviation == 'ISE-MASTER') {
+                    } else if($abbreviation == 'IS-MASTER' ) {
+                        $abbreviation = 'WI';
+                    } else if($abbreviation == 'ISE-MASTER' || $abbreviation == 'INF-MASTER') {
```

Comment:
```suggestion
                    } elseif($abbreviation == 'ISE-MASTER' || $abbreviation == 'INF-MASTER') {
```

Currently we are using `elseif` instead of `else if`